tpm2/constants.go

```diff
@@ -171,6 +187,13 @@ const (
 	CapabilityAuthPolicies
 )
 
+// Maximum valous for capabilities
+const (
+	MaxCapabilityBufferLength uint32 = 1024
```

Comment:
It looks like this "constant" isn't actually constant (which is my mistake). We don't need these constants after all, as the user can just pass `math.MaxUint32` for the number of capabilities (and `moreData` will then always be false as `MAX_CAP_BUFFER` has to be small enough to fit inside a normal TPM2 response).

Reply:
Removed these consts.